tests/Appsec/Mock.php

```diff
@@ -87,74 +88,132 @@ public function getEvents(array $names = [], array $addresses = [])
 }
 
 if (!function_exists('datadog\appsec\appsecMockEnabled')) {
-    function appsecMockEnabled() {
+    function appsecMockEnabled()
+    {
         return getenv('APPSEC_MOCK_ENABLED') === "true";
     }
 }
 
+if (!function_exists('datadog\appsec\track_user_login_success_event_automated')) {
+    /**
+     * This function is exposed by appsec but here we are mocking it for tests
+     */
+    function track_user_login_success_event_automated($userLogin, $userId, $metadata)
+    {
+        if (!appsecMockEnabled()) {
+            return;
+        }
+        $event = [
+            'userLogin' => $userLogin,
+            'userId' => $userId,
+            'metadata' => $metadata,
+
+        ];
+        AppsecStatus::getInstance()->addEvent($event, 'track_user_login_success_event');
+    }
+}
+
 if (!function_exists('datadog\appsec\track_user_login_success_event')) {
     /**
      * This function is exposed by appsec but here we are mocking it for tests
      */
-    function track_user_login_success_event($userId, $metadata, $automated) {
-        if(!appsecMockEnabled()) {
+    function track_user_login_success_event($userId, $metadata)
+    {
+        if (!appsecMockEnabled()) {
             return;
         }
         $event = [
             'userId' => $userId,
             'metadata' => $metadata,
-            'automated' => $automated
 
         ];
         AppsecStatus::getInstance()->addEvent($event, 'track_user_login_success_event');
     }
 }
 
-if (!function_exists('datadog\appsec\track_user_login_failure_event')) {
+if (!function_exists('datadog\appsec\track_user_login_failure_event_automated')) {
     /**
      * This function is exposed by appsec but here we are mocking it for tests
      */
-    function track_user_login_failure_event($userId, $exists, $metadata, $automated) {
-        if(!appsecMockEnabled()) {
+    function track_user_login_failure_event_automated($userLogin, $userId, $exists, $metadata)
+    {
+        if (!appsecMockEnabled()) {
             return;
         }
         $event = [
+            'userLogin' => $userLogin,
             'userId' => $userId,
             'exists' => $exists,
             'metadata' => $metadata,
-            'automated' => $automated
 
         ];
         AppsecStatus::getInstance()->addEvent($event, 'track_user_login_failure_event');
```

Comment:
The name here should be `track_user_login_failure_event_automated` instead of `track_user_login_failure_event`